### PR TITLE
Improve formatting in token related classes

### PR DESCRIPTION
**Pull request by lechu445**
Only cleanup. No functional changes.

### CODE REVIEW

**Review by jennyf19 — APPROVED** [review]
LGTM.

File: src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs [hunk 1]
```diff
@@ -41,13 +41,13 @@
...
     /// <summary>
-    /// A <see cref="SecurityTokenHandler"/> designed for creating and validating Json Web Tokens. 
+    /// A <see cref="SecurityTokenHandler"/> designed for creating and validating Json Web Tokens.
     /// See: http://tools.ietf.org/html/rfc7519 and http://www.rfc-editor.org/info/rfc7515.
     /// </summary>
     public class JsonWebTokenHandler : TokenHandler
     {
         /// <summary>
-        /// Gets the Base64Url encoded string representation of the following JWT header: 
+        /// Gets the Base64Url encoded string representation of the following JWT header:
```
Comment (brentschmaltz) [inline src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs:50]: Should be a period and the end instead of at ':'.

**Comment by brentschmaltz** [comment]
@lechu445 there are some conflicts here, you will need to rebase against dev.

File: src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs (hunk 1 above)
Comment (lechu445) [reply src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs:50]: I think this is not a typo but just a long text wrapped after ':' for better readability. See how Visual Studio renders it: 
image
Should I perform the change that you suggested anyway?

Comment (jennyf19) [reply src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs:50]: i would leave it as you have it @lechu445  w/the `:`

**Review by brentschmaltz — APPROVED** [review]
@lechu445 thanks again :-)

**Review by brentschmaltz — CHANGES_REQUESTED** [review]
@lechu445 thanks again :-)

Needs a rebase.

**Comment by lechu445** [comment]
@brentschmaltz I have rebased it onto the newest `brentsch/SystemTextJson`

**Comment by brentschmaltz** [comment]
this is a topic branch that i frequently rebase and squash.
not a good place to merge into.

**Pull request closed without merging**